libmscore/clef.cpp

```diff
@@ -148,6 +148,13 @@ void Clef::layout()
                         _clefTypes = staff()->clefType(0);
                   }
 
+            Measure* meas = clefSeg->measure();
+            if (meas && meas->system()) {
+                  const std::vector<MeasureBase*>& ml = meas->system()->measures();
+                  bool courtesy = tick == meas->endTick() && (meas == meas->system()->lastMeasure() || std::find(ml.begin(), ml.end(), meas) == ml.end());
```

Comment:
I had copied that line from 2.3. I did have to change it, though, because std::vector does not have indexOf(). Anyway, I added a pair of parentheses and extracted the std::find() expression to a local variable.